### ISSUE TITLE
TypeError: confirm is not a function

### ISSUE THREAD

**Issue by Grohden** [kettanaito/msw#2063]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 18 or higher

### Node.js version

v20.11.0

### Reproduction repository

https://github.com/Grohden/msw-cli-issue

### Reproduction steps

Just run `npx msw init ./app`, or just that command in a valid project

```
 msw-issue-cli git:(main) ✗ npx msw init ./app
Copying the worker script at "/Users/gabrielrohden/Desktop/Projects/Personal/msw-issue-cli/app"...

Worker script successfully copied!
  - ./app

Continue by describing the network in your application:
  

https://mswjs.io/docs/getting-started

      INFO In order to ease the future updates to the worker script,
      we recommend saving the path to the worker directory in your package.json.
msw init

Initializes Mock Service Worker at the specified directory

Positionals:
  publicDir  Relative path to the public directory                      [string]

Options:
  --version  Show version number                                       [boolean]
  --help     Show help                                                 [boolean]
  --save     Save the worker directory in your package.json            [boolean]
  --cwd      Custom current worker directory                            [string]

Examples:
  msw init
  msw init ./public
  msw init ./static --save

TypeError: confirm is not a function
    at promptWorkerDirectoryUpdate (/Users/gabrielrohden/.npm/_npx/7abdc5213ed84121/node_modules/msw/cli/init.js:183:10)
    at Object.init [as handler] (/Users/gabrielrohden/.npm/_npx/7abdc5213ed84121/node_modules/msw/cli/init.js:45:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

it seems that inquirer confirm needs to be imported with 
```
const confirm = require('@inquirer/confirm').default
```

 is my env with something wrong?

### Current behavior

Cli breaks trying to confirm something

### Expected behavior

Cli to not break trying to confirm something

**Comment by kettanaito** [comment]
Hi, @Grohden. Thanks for reporting this!

I can confirm this is an issue on the latest version of MSW that breaks the `npx msw init` command entirely. 

Not sure if the `.default` import is the right fix though. Need to look deeper into how `confirm` is bundled. From what I've seen, it's a CJS package.

**Comment by Grohden** [comment]
glad I can help a little 🙏 

I did the fix locally and at least that command asked for a confirmation for me

**Comment by serabariq** [comment]
Is there any solution about this? I have exact problem when doing it in nextjs 14.0.4 - react 18

**Comment by kkkhs** [comment]
i want to known how to fix it

**Comment by Grohden** [comment]
@serabariq @kkkhs specifically for init, just open the file from the stack trace, which is the init.js file:
```
TypeError: confirm is not a function
    at promptWorkerDirectoryUpdate (<path here>/node_modules/msw/cli/init.js:183:10)
    at Object.init [as handler] (<path here>/node_modules/msw/cli/init.js:45:7)
```

and change the inquirer import at the top of the file to have `.default` at the end:

```
const confirm = require('@inquirer/confirm').default
```

**Comment by alicenstar** [comment]
I'm also running into this issue, and it's a bit of a blocker. Would love for a fix to get prioritized.

**Comment by kettanaito** [comment]
## Released: v2.2.4 🎉

This has been released in v2.2.4!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.2.4)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.2.4)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
